sdk/dnsresolver/Microsoft.Azure.Management.DnsResolver/tests/Constants.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace DnsResolver.Tests
```

Comment:
fix the namespace (make it consistent in projects), also there is empty line in L6

Reply:
Resolved.